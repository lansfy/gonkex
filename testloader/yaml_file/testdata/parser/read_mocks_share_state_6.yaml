- name: first shared mocks
  method: GET
  path: /some/path1
  request: "requestbody"
  mocksParams:
    shareState: true
  mocks:
    someservice:
      strategy: nop

- name: first shared mocks
  method: GET
  path: /some/path2
  request: "requestbody"
  mocksParams:
    shareState: true

- name: second shared mocks
  method: GET
  path: /some/path3
  request: "requestbody"
  mocksParams:
    shareState: true
  mocks:
    someservice:
      strategy: nop

- name: second shared mocks
  method: GET
  path: /some/path4
  request: "requestbody"
  mocksParams:
    shareState: true
