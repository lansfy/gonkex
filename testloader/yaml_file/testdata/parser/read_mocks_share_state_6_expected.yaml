- GetName: first shared mocks
  GetMethod: GET
  Path: /some/path1
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_6.yaml
  GetLineNumber: 1
  FirstTestInFile: true
  ServiceMocksParams:
    SkipMocksResetAfterTest: true
  ServiceMocks:
    someservice:
      strategy: nop

- GetName: first shared mocks
  GetMethod: GET
  Path: /some/path2
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_6.yaml
  GetLineNumber: 11
  ServiceMocksParams:
    SkipMocksResetBeforeTest: true

- GetName: second shared mocks
  GetMethod: GET
  Path: /some/path3
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_6.yaml
  GetLineNumber: 18
  ServiceMocksParams:
    SkipMocksResetAfterTest: true
  ServiceMocks:
    someservice:
      strategy: nop

- GetName: second shared mocks
  GetMethod: GET
  Path: /some/path4
  GetRequest: requestbody
  GetFileName: testdata/parser/read_mocks_share_state_6.yaml
  GetLineNumber: 28
  LastTestInFile: true
  ServiceMocksParams:
    SkipMocksResetBeforeTest: true
